src/pixelator/graph/utils.py

```diff
@@ -303,6 +304,9 @@ def _edgelist_metrics_pandas_data_frame(
     metrics["b_pixel_count_per_a_pixel_stats"] = SummaryStatistics.from_series(
         edgelist.groupby("upia", observed=True)["upib"].nunique()
     )
+    metrics["a_pixel_count_per_b_pixel_stats"] = SummaryStatistics.from_series(
+        edgelist.groupby("upib", observed=True)["upia"].nunique()
```

Comment:
I suppose they are usually quite similar, but they are not the same. The metric that is based on `upia_per_upib` is now called `a_pixel_b_pixel_ratio_per_cell_stats`.

This metric actually captures the distribution of number of upias per unique upib over all components and is averaged over the number of unique upibs.

The `a_pixel_b_pixel_ratio_per_cell_stats` has the a to b pixel ratio in a component and is then averaged over all components.

Reply:
I think we maybe need to do these statistics in [`components_metrics`](https://github.com/PixelgenTechnologies/pixelator/blob/5e8d1518fd3c74b8bcc651517ff0026266a7cb22/src/pixelator/graph/utils.py#L59) instead so we have averages per cell instead of over all components. This will make the metrics more consistent and easier to understand.